mpt/core/src/main/java/org/apache/james/mpt/api/ImapFeatures.java

```diff
@@ -28,7 +28,10 @@
 public class ImapFeatures {
     
     public enum Feature {
-        NAMESPACE_SUPPORT
```

Comment:
We may define this in a _Capabilites_ class in james-mailbox-api project, and use it everywhere.